nav-app/src/app/datatable/data-table.component.ts

```diff
@@ -244,6 +246,25 @@ export class DataTableComponent implements AfterViewInit {
      */
     ngAfterViewInit(): void {
         // setTimeout(() => this.exportRender(), 500);
+        this.scrollRef.nativeElement.addEventListener('scroll', this.handleSidenavScroll);
+    }
+
+    ngOnDestroy() {
+        document.body.removeEventListener('scroll', this.handleSidenavScroll);
+    }
+
+    handleDescriptionDropdown() {
+        this.currentDropdown !== 'description' ? this.currentDropdown = 'description' : this.currentDropdown = '';
+        this.dropdownChange.emit(this.currentDropdown);
+    }
+
+    handleSidenavScroll = () => {
+        const tabHeight = 48;
+        const toolbarHeight = 34;
+        const footerHeight = 33;
+        const tabOffset = Math.floor(Math.min(Math.max(0, this.scrollRef.nativeElement.scrollTop/5), (tabHeight)));
```

Comment:
I think this 1:5 ratio of scrolling to tab-hiding is too high. Currently I have to be almost halfway down the page for the tabs to be fully hidden:

![image](https://user-images.githubusercontent.com/28868888/131857626-f0d4d56a-8e96-44f1-9259-003dc429cf3c.png)

In my comment on the prior PR I was concerned that 1:2 was even too high, and was suggesting making it 1:1 (removing the `/2` -- now `/5` -- entirely).

Reply:
With a 1:1 ratio you only need to scroll a little to recover the vertical height of the tabs: 
![image](https://user-images.githubusercontent.com/28868888/131858803-68febcdb-1c43-4439-acd7-814c8862b6e9.png)

Reply:
I was looking at the [material homepage](https://material.io/) and am considering a different approach, to detect scroll up from scroll down instead. How do you feel about this new behavior? This way, anytime the user scrolls up on the page it would just automatically show the header, instead of calculating how much of the header to show based on their scroll distance.